assets/src/blocks/Covers/CampaignCovers.js

```diff
@@ -1,23 +1,24 @@
+/* eslint-disable @wordpress/i18n-no-variables */
```

Comment:
What does this rule do? If I remove this line nothing seems to happen 🤔

Reply:
I get this error, that is why I added that exception to this file.

<img width="672" alt="Screenshot 2022-12-19 at 13 40 58" src="https://user-images.githubusercontent.com/38656549/208428797-d3da8aad-922d-4049-a92c-465c5b92f75e.png">